packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -474,19 +475,21 @@ export const setAccountMetadataKey =
             return account;
         }
 
+        const deviceMetaKey = device.metadata[encryptionVersion]?.key;
```

Comment:
nitpick: I would avoid abbreviations and prefer "metadata" instead of "meta" in all names